src/main/java/fr/xephi/authme/process/changepassword/AsyncChangePassword.java

```diff
@@ -84,7 +87,9 @@ public void changePasswordAsAdmin(CommandSender sender, final String playerName,
 
         HashedPassword hashedPassword = passwordSecurity.computeHash(newPassword, lowerCaseName);
         if (dataSource.updatePassword(lowerCaseName, hashedPassword)) {
-            bungeeSender.sendAuthMeBungeecordMessage(MessageType.REFRESH_PASSWORD, lowerCaseName);
+            if (dataSource instanceof CacheDataSource) {
+                bungeeSender.sendAuthMeBungeecordMessage(MessageType.REFRESH_PASSWORD, lowerCaseName);
+            }
```

Comment:
1) Maybe it‘s more elegant to have the bungee sender take care of checking that if statement. The method could be named accordingly. 

2) Can we introduce a method on DataSource for this? Something like `boolean isCached()` or `boolean hasCache()`